api/collections/views.py

```diff
@@ -194,7 +206,7 @@ def perform_destroy(self, instance):
         instance.delete()
 
 
-class CollectionDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, CollectionMixin):
+class CollectionDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, CollectionMixin, CollectionPreprintsMixin):
```

Comment:
Does `CollectionDetail` need `CollectionPreprintsMixin`? I can't see where `collection_preprints()` is used here or in a `CollectionDetail` subclass.

Reply:
Ah nvm, it's used in the `CollectionDetailSerializer`

Reply:
(not a blocker) What do you think about adding the `collection_preprints()` to the `CollectionMixin` instead of creating another mixin? In theory `CollectionMixin` could also have `collection_nodes()` and `collection_registrations()` helpers

Reply:
good idea 👍